### PR TITLE
Update 2.graph-modeling.md

### PR DESCRIPTION
**Pull request by foesa-yang**

### CODE REVIEW

**Review by whitewum — APPROVED** [review]

File: docs-2.0/8.service-tuning/2.graph-modeling.md [hunk 1]
```diff
@@ -68,6 +68,48 @@ nebula> MATCH (v1:player)-[e:temp]->() return collect(e.tmp);
...
+1. Insert edges with [INSERT EDGE](../3.ngql-guide/13.edge-statements/1.insert-edge.md) statement before the source vertex or the destination vertex exists.
+
+2. Delete vertices with [DELETE VERTEX](../3.ngql-guide/12.vertex-statements/4.delete-vertex.md) statement and the `WITH EDGE` option is not used. At this time, the system does not delete the related outgoing and incoming edges of the vertices. There will be dangling edges by default.
+
+Dangling edges may appear in Nebula Graph {{nebula.release}} as the design. And there is no MERGE statements of openCypher. The guarantee for dangling edges depends entirely on the application level. You can use [GO](../3.ngql-guide/7.general-query-statements/3.go.md) and [LOOKUP](../3.ngql-guide/7.general-query-statements/5.lookup.md) statements to find a dangling edge, but cannot use the [MATCH](../3.ngql-guide/7.general-query-statements/2.match.md) statement to find a dangling edge.
+
+Examples:
+
+```bash
+// Insert source vertex is '11', destination vertex is '13' and neither exists in the database
```
Comment (abby-cyber) [inline docs-2.0/8.service-tuning/2.graph-modeling.md:86]: Insert an edge that connects two vertices, with one vertex's ID being '11' and the other vertex's ID being '13', and the two vertices do not exist in the graph.

File: docs-2.0/8.service-tuning/2.graph-modeling.md [hunk 2]
```diff
@@ -68,6 +68,48 @@ nebula> MATCH (v1:player)-[e:temp]->() return collect(e.tmp);
...
+1. Insert edges with [INSERT EDGE](../3.ngql-guide/13.edge-statements/1.insert-edge.md) statement before the source vertex or the destination vertex exists.
+
+2. Delete vertices with [DELETE VERTEX](../3.ngql-guide/12.vertex-statements/4.delete-vertex.md) statement and the `WITH EDGE` option is not used. At this time, the system does not delete the related outgoing and incoming edges of the vertices. There will be dangling edges by default.
+
+Dangling edges may appear in Nebula Graph {{nebula.release}} as the design. And there is no MERGE statements of openCypher. The guarantee for dangling edges depends entirely on the application level. You can use [GO](../3.ngql-guide/7.general-query-statements/3.go.md) and [LOOKUP](../3.ngql-guide/7.general-query-statements/5.lookup.md) statements to find a dangling edge, but cannot use the [MATCH](../3.ngql-guide/7.general-query-statements/2.match.md) statement to find a dangling edge.
+
+Examples:
+
+```bash
+// Insert an edge that connects two vertices, with the source vertex's ID being '11' and the destination vertex's ID being '13', and the two vertices do not exist in the graph.
```
Comment (whitewum) [inline docs-2.0/8.service-tuning/2.graph-modeling.md:86]: this line is tooooooooo looooonnnnnggggg

Comment (abby-cyber) [inline docs-2.0/8.service-tuning/2.graph-modeling.md:86]:
```suggestion
// Insert an edge that connects two vertices that do not exist in the graph. The source vertex's ID is '11' and the destination vertex's ID is'13'. 
```

File: docs-2.0/8.service-tuning/2.graph-modeling.md [hunk 3]
```diff
@@ -68,6 +68,54 @@ nebula> MATCH (v1:player)-[e:temp]->() return collect(e.tmp);
...
+
+A dangling edge is an edge that only connects to a single vertex and only one part of the edge connects to the vertex.
+
+In Nebula Graph {{ nebula.release }}, dangling edges may appear in the following two cases.
+
+1. Insert edges with [INSERT EDGE](../3.ngql-guide/13.edge-statements/1.insert-edge.md) statement before the source vertex or the destination vertex exists.
+
+2. Delete vertices with [DELETE VERTEX](../3.ngql-guide/12.vertex-statements/4.delete-vertex.md) statement and the `WITH EDGE` option is not used. At this time, the system does not delete the related outgoing and incoming edges of the vertices. There will be dangling edges by default.
+
+Dangling edges may appear in Nebula Graph {{nebula.release}} as the design. And there is no MERGE statements of openCypher. The guarantee for dangling edges depends entirely on the application level. You can use [GO](../3.ngql-guide/7.general-query-statements/3.go.md) and [LOOKUP](../3.ngql-guide/7.general-query-statements/5.lookup.md) statements to find a dangling edge, but cannot use the [MATCH](../3.ngql-guide/7.general-query-statements/2.match.md) statement to find a dangling edge.
```
Comment (randomJoe211) [inline docs-2.0/8.service-tuning/2.graph-modeling.md:81]:
```suggestion
Dangling edges may appear in Nebula Graph {{nebula.release}} as the design allow it to exist. And there is no MERGE statement like openCypher has. The existence of dangling edges depends entirely on the application level. You can use [GO](../3.ngql-guide/7.general-query-statements/3.go.md) and [LOOKUP](../3.ngql-guide/7.general-query-statements/5.lookup.md) statements to find a dangling edge, but cannot use the [MATCH](../3.ngql-guide/7.general-query-statements/2.match.md) statement to find a dangling edge.
```

**Review by abby-cyber — APPROVED** [review]

**Review by cooper-lzy — APPROVED** [review]

**Pull request merged**